templates/solutions/automation/index.html

```diff
@@ -88,26 +88,24 @@ <h3 class="p-heading--5">Trusted and transparent</h3>
     {%- endif -%}
 
     {%- if slot == 'list_item_description_3' -%}
-      <div class="p-section--shallow">
-        <p>
-          Unlike most automation tools, our software, automations, and orchestrators are open source &mdash; giving you transparency and confidence to operate.
-        </p>
-      </div>
+      <p>
+        Unlike most automation tools, our software, automations, and orchestrators are open source &mdash; giving you transparency and confidence to operate.
+      </p>
     {%- endif -%}
   {%- endcall -%}
 
   <div class="p-section u-fixed-width u-no-padding--top">
     <div class="grid-row">
       <div class="grid-col-6 grid-col-medium-4 grid-col-start-large-3">
-        <div class="p-image-container--cinematic-on-large is-highlighted p-image-container--square-on-small"
+        <div class="p-image-container--16-9 is-highlighted p-image-container--square-on-small"
              style="padding: 1.5rem">
           {{ image(url="https://assets.ubuntu.com/v1/72910538-72910538d75908ad5d123f0c8548d6f81ee51dc3.png",
                     alt="",
                     width="1068",
                     height="864",
                     hi_def=True,
                     loading="lazy",
-                    attrs={"class": "p-image-container__image"}) | safe
+                    attrs={"class": "p-image-container__image", "style": "padding: 2rem 0"}) | safe
```

Comment:
Maybe this is the intention, but right at the point where this gets resized to mobile the padding only appears at the top and bottom of the image and seems a bit large in relation to the image, but it's fine if that's how it's supposed to be.

<img width="621" height="661" alt="Image" src="https://github.com/user-attachments/assets/e8b03e74-1a71-4c21-85a8-0d7f08be26cc" />